Exercise_7/scripts/ticTacToe/KeyActions.js

```diff
@@ -0,0 +1,42 @@
+import {SPACE_KEY, LEFT_KEY, RIGHT_KEY, UP_KEY, DOWN_KEY} from './Constraints';
+
+const KeyActions = () => {
+    let selectedFieldIndex = 4;
+    
+    const keyMoves = {
+        [SPACE_KEY]: action => action(selectedFieldIndex),
+
+        [LEFT_KEY]: action => {
+            selectedFieldIndex = selectedFieldIndex >= 1 ? --selectedFieldIndex : 8;
```

Comment:
This is how you must not do. Let's go step by step.
If `selectedFieldIndex >= 1` you mutate variable through decremention `--selectedFieldIndex` and then return new value of it to the same `selectedFieldIndex` variable. Replace `--selectedFieldIndex` by `selectedFieldIndex - 1`.